MAUI/Themes/Themes.md

```diff
@@ -0,0 +1,126 @@
+             x:Class="SampleBrowser.Maui.App"
+             ...>
+<Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <!-- Theme resource dictionary -->
+                <syncTheme:SyncfusionThemeResourceDictionary SfVisualTheme="MaterialDark"/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+</Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+
+![DarkThemeImage](Images/DarkTheme.png)
+
+## Overriding the default theme
+
+The theme resource dictionary contains a set of keys that are mapped to the style in control style dictionaries. The default appearance of themes can be customized by overriding the key values.
+
+The following section explains how to override both the primary and control specific keys.
+
+### Overriding the control specific keys
+
+In addition to the primary keys, the theme resource dictionary also contains the keys that are specific to each controls; these keys can also be overridden. You can find the keys and the UI elements to which they are mapped to all the controls in this `documentation`.
+
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+<Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary SfVisualTheme="MaterialDark"/>
+            <ResourceDictionary>
+                <Color x:Key="SfRadialGaugeNeedlePointerNeedleFillColor">LimeGreen</Color>
+                <Color x:Key="SfRadialGaugeNeedlePointerKnobFillColor">LimeGreen</Color>
+                <Color x:Key="SfRadialGaugeNeedlePointerTailFillColor">Black</Color>
+                <Color x:Key="SfRadialGaugeAxisLineFillColor">Green</Color>
+                <Color x:Key="SfRadialGaugeRangePointerFillColor">LimeGreen</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+</Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+
+![OverridingControlSpecificColors](Images/DarkTheme_ControlSpecific.png)
+
+## Creating your own theme
+
+As an alternative approach to the above methods, you can also create your own theme. To create own theme, first, you need to merge the resource, whose key name should be "ControlName" + "Theme" based on the controls, e.g., SfChartTheme and SfTextInputLayoutTheme. You can find this key for each control in this `documentation` . After merging this resource, you need to merge the required color resources based on the UI elements that need to be customized. You can find the keys and the UI elements to which they are mapped to all the controls in this `documentation`.
+Using this approach, you can create your own theme to all the controls or for specific controls you need. 
```

Comment:
Using this approach, you can create your own theme for all the controls or specific controls you need.

Reply:
Changes addressed.